src/engine.rs

```diff
@@ -94,7 +94,7 @@ impl Engine for PbftEngine {
                 // ViewChange if necessary
                 if node.check_faulty_primary_timeout_expired(state) {
                     warn!("Faulty primary timeout expired; proposing view change");
-                    handle_pbft_result(node.start_view_change(state, state.view + 1));
+                    log_any_error(node.start_view_change(state, state.view + 1));
```

Comment:
These log_any_error() calls seem a bit awkward. In any case, this might be a good use for a macro.

Reply:
Could you explain what makes this a good use for a macro? Or do you have other suggestions for how to make this less awkward? The only thing I can think of at the moment would be to just call `.unwrap_or_else(|err| error!("{}", err)` in these situations.